src/segment_python_test.go

```diff
@@ -83,6 +83,6 @@ func TestPythonPyEnvWithVersion(t *testing.T) {
 	}
 	python := bootStrapPythonTest(args)
 	assert.True(t, python.enabled())
-	assert.Equal(t, "3.8.4", python.language.version)
 	assert.Equal(t, expected, python.string())
+	assert.Equal(t, "3.8.4", python.language.version)
```

Comment:
Since we get the version in the `string()` function I had to swap the lines. No behaviour change.